test/api/v3/integration/tasks/groups/GET-approvals_group_id.test.js

```diff
@@ -34,7 +34,12 @@ describe('GET /approvals/group/:groupId', () => {
 
     let memberTasks = await member.get('/tasks/user');
     syncedTask = find(memberTasks, findAssignedTask);
-    await member.post(`/tasks/${syncedTask._id}/score/up`);
+    await expect(member.post(`/tasks/${syncedTask._id}/score/up`))
```

Comment:
Ehh. I was catching the thrown error this way. I will switch it to a try..catch.